src/app/enums/docker-nvidia-status.enum.ts

```diff
@@ -0,0 +1,17 @@
+import { marker as T } from '@biesbjerg/ngx-translate-extract-marker';
+
+export enum DockerNvidiaStatus {
```

Comment:
Should part of a different PR because we would need to backport it.

Reply:
Wrong branch. Changes were reverted